orchagent/portsorch.h

```diff
@@ -266,6 +268,7 @@ class PortsOrch : public Orch, public Subject
     void doLagMemberTask(Consumer &consumer);
 
     void doTask(NotificationConsumer &consumer);
+    void doTask(swss::SelectableTimer &timer);
```

Comment:
In that case, this will become an infra. change to the orchagent, and is outside the scope of this PR, it's better to revisit the timer support in the orchagent if this is a concern